src/mod11.js

```diff
@@ -8,12 +8,20 @@ function gen(n) {
 /**
  * NOTE: caller of this function must handle the case where this function returns 10.
  *       typically 10 should be '-' when used as control digit.
```

Comment:
This note is made obsolete by this PR, it should be removed:

```suggestion
```

Reply:
Fixed by c240816